### PR TITLE
Fix UI errors in the console

### PR DESCRIPTION
**Pull request by DonOmalVindula**
### Purpose
$subject

### Related Issues
- https://github.com/wso2/product-is/issues/16384

### Related PRs
- N/A

### Checklist
- [x] e2e cypress tests locally verified.
- [x] Manual test round performed and verified.
- [x] UX/UI review done on the final implementation.
- [x] Documentation provided. (Add links if there are any)
- [x] Unit tests provided. (Add links if there are any)
- [x] Integration tests provided. (Add links if there are any)

### Security checks
- [x] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines?
- [x] Ran FindSecurityBugs plugin and verified report?
- [x] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

File: apps/console/src/extensions/components/users/api/organization.ts [hunk 1]
```diff
@@ -1,15 +1,24 @@
 /**
- * Copyright (c) 2022, WSO2 LLC. (http://www.wso2.com). All Rights Reserved.
+ * Copyright (c) 2022, WSO2 LLC. (https://www.wso2.com). All Rights Reserved.
```
Comment (NipuniBhagya) [inline apps/console/src/extensions/components/users/api/organization.ts:2]:
```suggestion
 * Copyright (c) 2022-2023, WSO2 LLC. (https://www.wso2.com). All Rights Reserved.
```

File: apps/console/src/features/users/components/wizard/add-user-wizard.tsx [hunk 2]
```diff
@@ -420,6 +425,8 @@ export const AddUserWizard: FunctionComponent<AddUserWizardPropsInterface> = (
      * This function handles adding the user.
      */
     const addUserBasic = (userInfo: AddUserWizardStateInterface) => {
+        console.log("userInfo", userInfo);
```
Comment (NipuniBhagya) [inline apps/console/src/features/users/components/wizard/add-user-wizard.tsx:428]:
```suggestion
```

**Review by NipuniBhagya — APPROVED** [review]

File: apps/console/src/features/users/components/wizard/add-user-wizard.tsx (hunk 2 above)
Comment (DonOmalVindula) [reply apps/console/src/features/users/components/wizard/add-user-wizard.tsx]: Fixed with 44fe2a53dea4d5592734ed150a2fa6ef1e116dcf

File: apps/console/src/extensions/components/users/api/organization.ts (hunk 1 above)
Comment (DonOmalVindula) [reply apps/console/src/extensions/components/users/api/organization.ts]: Fixed with 44fe2a53dea4d5592734ed150a2fa6ef1e116dcf

**Pull request merged**